projects/cli/src/templates/_elm-land/server/main.js

```diff
@@ -1,132 +1,55 @@
-import { Elm } from '../src/Main.elm'
```

Comment:
This file is just removing the Elm import (to know be handled via elm-watch), and removing the hot module code, also to be handled by elm-watch.